pages/about.vue

```diff
@@ -44,7 +44,7 @@
       </v-row>
 
       <!-- We value row -->
-      <v-row :class="pageSectionClass" id="we-value">
+      <v-row :class="{[pageSectionClass]: true, 'mx-2': $vuetify.breakpoint.smAndDown}" id="we-value">
```

Comment:
I wonder- if `pageSectionClass` isn't enough to make this look good, should we be adding this margin + breakpoint to `pageSectionClass`?

Reply:
Yeah I see what you mean, it may be the other pages that use `pageSectionClass` aren't wide enough to have the issue. I guess we could try updating `pageSectionClass` and see what happens...

Reply:
@alexanian I tried adding this class to `pageSectionClass` but I couldn't figure out how to access `vuetify.breakpoint.smAndDown` in the mixin js file. I tried various vuetify imports to no avail. Do you have any suggestions?

Reply:
Would something [like this](https://github.com/covid19risk/website/pull/110/files#diff-c1d6953f4b3b2c866d028672b2c4e439R63-R68) work?

```scss
  @import "~vuetify/src/styles/settings/_variables";

  #filter-group {
    @media #{map-get($display-breakpoints, 'sm-and-down')} {
      flex-direction: column;
    } 
  }
```

Reply:
Ah nevermind I got it! It just needed to be a computed property in the `spacing-mixins.js` file